include/crm/common/xml_compat.h

```diff
@@ -192,6 +192,9 @@ void expand_plus_plus(xmlNode * target, const char *name, const char *value);
 //! \deprecated Do not use
 void crm_xml_init(void);
 
+//! \deprecated Do not use
```

Comment:
Recommend exiting with `crm_exit()` instead. (We should probably separate cleanup from exiting.)